AUTHORS

```diff
@@ -6,14 +6,14 @@ Authors with Contributions in the Public Domain:
 Lewis Rossman <LRossman@cinci.rr.com>
 Robert Dickinson <robert.dickinson@gmail.com>
 Michael Tryby <tryby.michael@epa.gov>
+Katherine Ratliff <ratliff.katherine@epa.gov>
```

Comment:
This is a silly discussion point but I think it's probably important. I'm thinking we should revisit this file and order the Authors based on first contribution date and use an asterisk and a foot note to denote whether a contribution is under BSD2 or public domain.  What are your thoughts?

Reply:
@bemcdonnell That sounds good to me!

Reply:
@katmratliff, do you mind adding those changes to this PR?

Reply:
@bemcdonnell please make sure that I ordered authors correctly, and let me know if you'd like any of the wording changed.